site-docs/src/main/paradox/dynamodb/usage/index.md

```diff
@@ -0,0 +1,42 @@
+# DynamoDB Client
+
+The client constructor takes a `DynamoDbAsyncClient` object. The default `apply` method passes one in with sensible defaults but you can pass your own in if necessary.
+
+```scala
+val dynamoDbAsyncClient: DynamoDbAsyncClient = ???
+val clientWithCustom = new DADynamoDBClient(dynamoDbAsyncClient)
+
+val clientWithDefault = DADynamoDBClient()
+```
+
+The client exposes two methods:
+
+```scala
+def getItem(dynamoDbRequest: DynamoDbRequest): F[AttributeValue]
+
+def updateItem(dynamoDbRequest: DynamoDbRequest): F[Int]
+```
+
+The method takes a dynamoDbRequest Case Class:
+
+```scala
+  case class DynamoDbRequest(tableName: String,
+                             primaryKeyAndItsValue: Map[String, AttributeValue],
+                             itemName: String,
+                             itemValueToUpdate: Option[AttributeValue]=None)
+```
+consisting of:
+- tableName - the DynamoDB table you want to make the edits in
+- primaryKeyAndItsValue - the primary key and the value to filter it down
+  - you can use the .builder and pass the value to the 'type' method ('.s', '.b', '.n' etc)
+  - *Warning* adding a value that isn't already present in the table will cause DynamoDb to add another row so watch out for misspellings
+- itemName - the item name you want the value of
```

Comment:
Should `item` be `attribute` throughout this libary? Dynamo's terminology is an item=row and an attribute=column.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithItems.html